plugins/doc_fragments/eos.py

```diff
@@ -11,6 +11,14 @@ class ModuleDocFragment(object):
 
     # Standard files documentation fragment
     DOCUMENTATION = r"""options:
+  version:
```

Comment:
I don't think this is where this parameter should be. For one, this will only apply to old-style modules, as you can see from which modules' documentation changed. But I also don't see what the point would be of changing this on a module that is not returning the response directly to the user. If there is enough of a difference between the response with "latest" and some other version that we would want to switch to a different version, I would assume that we would need to go into the module to fix that incompatibility anyway.

For that matter, I'm not actually seeing anywhere outside of `eos_command` where you are using the value of version` anyway, though I might be missing that.

This should actually _probably_ live in the command parameter spec that we... uh don't actually document anywhere. Ideally it would get slipped in after https://github.com/ansible-collections/arista.eos/pull/280/files#diff-4ad4981a62c8fb377775d02939951f59d3a7a74b0732dd4a17e52dfaf510d5afR57-R61 and then we can replace the current `transform_commands` imported from netcommon with an eos-specific one that includes `version` and drop the rest of the changes in eos_command.

Hopefully this all makes some kind of sense? I'm happy to clarify any of this if you need.

Reply:
I have made the necessary changes to include the 'version' parameter in eos_command module.